recipe/conda_build_config.yaml

```diff
@@ -263,7 +263,7 @@ flann:
 fontconfig:
   - 2.13
 freetype:
-  - 2.8.1
+  - 2.9.1
```

Comment:
This doesn't have to be changed if the pin is major

Reply:
My question there is: there are new symbols and maybe some libraries will need them at build time? (I don't know about freetype but I've seen this with netcdf in the past.)

Reply:
I don't know. we use an older numpy version and if the library needs newer version, it is done in the recipe right?

Reply:
> I don't know. we use an older numpy version and if the library needs newer version, it is done in the recipe right?

Numpy is closer to us and easier to identify things like that. Other libraries I have no idea and I don't really inspect the logs to check if there are warnings and/or missing features. My policy is to update the minimum every now and then. Also, because this was already pinned we don't need to rush things and start rebuilding everything.

However, I don't have a strong opinion on this and I can revert if you disagree.

Reply:
I'm fine with either way.

Reply:
OK. Let's go for "new is better" :smile: